app/src/componentLibrary/modal/modalLayout/modalLayout.scss

```diff
@@ -96,3 +96,8 @@ $WIDTH-LARGE: 720px;
 .size-large {
   @include modalWidth($WIDTH-LARGE);
 }
+
+.overlay-light-cyan {
+  background-color: rgba($COLOR--light-cyan, 0.7);
+  z-index: $Z_INDEX-GRID_HEADER;
```

Comment:
It's confusing because modal layout depends on var that belongs to grid. maybe better create another one variable